File: core/src/ten_runtime/binding/python/interface/ten/log_level.py
#
# Copyright © 2024 Agora
# This file is part of TEN Framework, an open source project.
# Licensed under the Apache License, Version 2.0, with certain conditions.
# Refer to https://github.com/TEN-framework/ten_framework/LICENSE for more
# information.
#
from enum import IntEnum


class LogLevel(IntEnum):
    INVALID = 0
    VERBOSE = 1
    DEBUG = 2
    INFO = 3
    WARN = 4
    ERROR = 5
    FATAL = 6

File: core/src/ten_runtime/binding/go/interface/ten/log_level.go
//
// Copyright © 2024 Agora
// This file is part of TEN Framework, an open source project.
// Licensed under the Apache License, Version 2.0, with certain conditions.
// Refer to https://github.com/TEN-framework/ten_framework/LICENSE for more
// information.
//

package ten

type LogLevel int32

const (
	LogLevelVerbose = 1
	LogLevelDebug   = 2
	LogLevelInfo    = 3
	LogLevelWarn    = 4
	LogLevelError   = 5
	LogLevelFatal   = 6
)
